components/admin_console/manage_teams_modal/manage_teams_dropdown.jsx

```diff
@@ -44,10 +44,17 @@ export default class ManageTeamsDropdown extends React.Component {
 
     render() {
         const isTeamAdmin = Utils.isAdmin(this.props.teamMember.roles) || this.props.teamMember.scheme_admin;
+        const isSysAdmin = Utils.isSystemAdmin(this.props.user.roles);
+        const isGuest = Utils.isGuest(this.props.user) || this.props.teamMember.scheme_guest;
```

Comment:
I think is not needed the `this.props.teamMember.scheme_guest`, because a guest user is a guest at system wide, so if the user is a guest, no matter what the team members says.